website/markdown/posts/2020-02-26-version-1.3.0/post.mdx

```diff
@@ -0,0 +1,90 @@
+author: pepibumur
+---
+
+Tuist 1.3.0 has just [been published](https://github.com/tuist/tuist/releases/tag/1.3.0).
+In this blog post,
+I'll walk you through the improvements and fixes that we shipped in this release,
+and hint what are the upcoming features that we plan to work on.
+
+## Generate the projects when editing the project in Xcode
+
+Tuist has a command,
+`tuist edit`,
+which creates an Xcode project to edit your project manifest files.
+The benefit of that is that developers can get inline documentation and validate the syntax by just building the manifest in Xcode.
+We know this feature is quite handy,
+so [we improved](https://github.com/tuist/tuist/pull/958) it by adding support for running `tuist generate` directly from Xcode.
+The editable Xcode project will have an extra scheme that runs the generation command.
+That way users don't have to go back to the terminal and run `tuist generate` manually.
+
+## Preserve user data when generating the projects
+
+Xcode projects have files of type `.xcuserdata` that Xcode generates with user-specific configuration.
+Those files are deleted when projects are generated with `tuist generate`.
+Thanks to [this work](https://github.com/tuist/tuist/pull/1006) that's not the case anymore.
+From Tuist 1.3.0,
+`tuist generate` generates the projects preserving that data.
+
+## Performance-related improvements
+
+The generation of Xcode project **needs to be fast.**
+Unfortunately, the performance of the generation logic is something that we have disregarded since the inception of the project, and now some users are hitting slow project generations.
+We decided it's time to tackle this so the first step that we took was adding a [helper utility](https://github.com/tuist/tuist/pull/957) that helps us benchmark the performance.
+Thanks to it,
+we'll be able to uncover performance bottlenecks and fail pull requests if we introduce regressions.
+
+Moreover, we made [some improvements](https://github.com/tuist/tuist/pull/980) in the logic that sorts the project groups and files getting an improvement of roughly 50%.
+
+## Project manifesto
+
+As the project grows,
+it's crucial for users, contributors, and maintainers to know what are our design pillars.
+For that reason,
+we updated our documentation to include a [manifesto](/docs/contribution/manifesto/) that contains a set of principles that are foundational to the design and development of Tuist.
+They help align users and contributors with the vision of the project.
+
+## Installation from Google Cloud Storage
+
+Tuist used to install new versions of Tuist by pulling the binaries from the GitHub releases.
+As soon as users reported, that caused some issues because GitHub's API returned limit errors.
```

Comment:
`As soon as users reported, that caused...` Maybe rather something like: But we have got reports from users that this has caused ... ? The sentence, as it is written now, does not make sense, I think

Reply:
You are right. It should be:

```
As users reported that caused some...
```